src/components/NavigationBar.tsx

```diff
@@ -41,30 +41,25 @@ export default function NavigationBar() {
                   {/* Current: "border-indigo-500 text-gray-900", Default: "border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-700" */}
                   <a
                     href="#"
+                    onClick={() => "window.scrollTo(0,0)"}
```

Comment:
This is necessary in this case?? The `href= '#'` do it I think.

Reply:
Oh, I didn't realize that! Removed :)